scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Conversion.scala

```diff
@@ -68,11 +69,15 @@ object ToBigDecimalConversion extends ToNumericConversion[JBigDecimal] {
 
   override def convertEither(value: Any): Either[Throwable, JBigDecimal] =
     value match {
-      case v: JBigDecimal => Right(v)
-      case v: JBigInteger => Right(new JBigDecimal(v))
+      case v: JBigDecimal => Right(v.setScale(Math.max(v.scale(), NumberUtilsConsts.DEFAULT_BIG_DECIMAL_SCALE)))
```

Comment:
Let's extract:

```
v.setScale(Math.max(v.scale(), NumberUtilsConsts.DEFAULT_BIG_DECIMAL_SCALE))
```

to separated method.

Reply:
done